doc/content/microreactors/gcmr/GCMR_Core_Neutronics.md

```diff
@@ -0,0 +1,100 @@
+# Neutronic Modeling of the Whole Core Gas-Cooled Microreactor (GCMR)
+
+## GCMR Core Description
+
+The GC-MR, developed at ANL, serves as a modeling experiment to explore design options considered by microreactor vendors, encompassing features like control drums, hydride metal, and TRISO fuel. This horizontal gas-cooled microreactor system boasts a thermal power of 20 MW and an approximate lifespan of 9.5 years. Its power conversion cycle utilizes a Brayton cycle, circulating high-temperature (650°C-850°C) and high-pressure (7 MPa) helium coolant. Surrounding the core are BeO radial and axial neutron reflectors, with twelve control drums positioned in the reflector encircling the core. These control rods, containing 96%-enriched B4C, are inserted into holes within the middle core assemblies. Displayed in [Fig_1] and [Fig_2]  are radial and axial views of the core, which is relatively compact, measuring 2.42 m in diameter and 2.40 m in length.
+
+!media media/gcmr/core/Fig_1.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_1
+      caption= Radial View of the GCMR core
+
+!media media/gcmr/core/Fig_2.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_2
+      caption= Axial view of the GCMR core
+
+The core comprises three types of fuel assemblies: Assembly A in the inner region, Assembly B in the middle, and Assembly C in the outer core region. Each fuel assembly incorporates TRISO fuel blocks containing 19.75 wt% of LEU fuel and Yttrium hydride moderator pins encased in FeCrAl envelopes. Additionally, burnable poison blocks, composed of Gd2O3 particles with a 25% packing fraction distributed axially, and Helium coolant channels are integrated. Assembly A's detailed design is provided in [Fig_3], while [Fig_4] illustrates the design differences among the three assemblies. Notably, Assemblies B and C share similarities, except for Assembly B's inclusion of a core center location for a control rod. Key design parameters of the GCMR core are:
+
+!media media/gcmr/core/Fig_3.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_3
+      caption= GCMR assembly of Type A
+
+!media media/gcmr/core/Fig_4.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_4
+      caption= Design of the three types of fuel assemblies in the core
+
+| Parameter (unit)| Value |
+| - | - |
+| Reactor Power (MWth) | 20.0 |
+| Core diameter (cm) | 242.0 |
+| Core height (cm) | 240.0 |
+| Active height (cm) | 200.0 |
+| Different core zones | 3 |
```

Comment:
Radial or axial? do you want to say enrichment zones?

Reply:
Clarified. 

We have 3 radial zones ( different FA designs )